scavenger-api/src/main/kotlin/com/navercorp/scavenger/repository/SnapshotNodeDao.kt

```diff
@@ -5,28 +5,46 @@ import com.navercorp.scavenger.entity.SnapshotNodeEntity
 import com.navercorp.scavenger.repository.sql.SnapshotNodeSql
 import com.navercorp.spring.data.jdbc.plus.sql.provider.EntityJdbcProvider
 import com.navercorp.spring.data.jdbc.plus.sql.support.JdbcDaoSupport
+import com.navercorp.spring.data.jdbc.plus.sql.support.trait.SingleValueSelectTrait
 import org.springframework.stereotype.Repository
 
 @Repository
 class SnapshotNodeDao(
     entityJdbcProvider: EntityJdbcProvider,
     snapshotNodeRepository: SnapshotNodeRepository
-) : JdbcDaoSupport(entityJdbcProvider), SnapshotNodeRepository by snapshotNodeRepository {
+) : JdbcDaoSupport(entityJdbcProvider), SingleValueSelectTrait, SnapshotNodeRepository by snapshotNodeRepository {
     private val sql: SnapshotNodeSql = super.sqls(::SnapshotNodeSql)
 
-    fun findAllExportSnapshotNode(
+    fun selectAllExportSnapshotNode(
```

Comment:
we use find here.

Reply:
fixed in 90c8f8a39bdbfcbafd0070c2c7150a5cc9821932